### ISSUE TITLE
Incorrect colors even with minimal setup

### ISSUE THREAD

**Issue by crondinini** [mhartington/oceanic-next#65]
# Problems summary
1. Some of the colors that appear are different from the screenshots of them.  
2. I'm getting the following error when using your airlines themes instruction
`Could not resolve airline theme "oceanic-next". Themes have been migrated to github.com/vim-airline/vim-airline-themes.`

## Expected
Colors to be correct. For example, from the screenshot below: 
- `clientId` to be blue 
- semi-colon to be red 
- `var` to be red
- background color seems to be different 

## Environment Information

 * terminal: iTerm2 Build 3.1.5
 * vim/nvim version: NVIM v0.2.2
 * mac OS: High Sierra 10.13.2

## Provide a minimal vim rc with less than 50 lines 

```vim
call plug#begin('~/.config/nvim/plugged')
Plug 'mhartington/oceanic-next'
Plug 'othree/yajs.vim'
Plug 'vim-airline/vim-airline'
call plug#end()

syntax enable
let $NVIM_TUI_ENABLE_TRUE_COLOR=1
colorscheme OceanicNext
set background=dark
let g:airline_theme='oceanic-next'
```

## The reproduce ways from neovim starting

 1. I'm simply opening the file with `nvim test.js`

## Screen shot (if possible)
Using test.js
![image](https://user-images.githubusercontent.com/21973269/35647175-fbe5da9a-06d1-11e8-9b42-a024c4dd786f.png)

**Comment by mhartington** [comment]
Can you try adding this to your init.vim

```
" For Neovim 0.1.3 and 0.1.4
let $NVIM_TUI_ENABLE_TRUE_COLOR=1

" Or if you have Neovim >= 0.1.5
if (has("termguicolors"))
 set termguicolors
endif
```

The env variable form of true color support is no longer supported in recent builds of neovim

**Comment by crondinini** [comment]
Oh right! I had tried that before with my longer setup and it was messing the colors up. I added it now again with the minimal setup and it worked. Sorry for the oversight, should have tried that again before opening the issue. 

In case it might be useful for you to know, I realized that `set background=dark` is what was messing up the theme. You're probably aware of that, but here's the screenshot of my config file and test.js with that enabled: 

![image](https://user-images.githubusercontent.com/21973269/35770265-27e7f798-0918-11e8-98ab-a125356abccb.png)

and 

![image](https://user-images.githubusercontent.com/21973269/35770272-45b5f798-0918-11e8-8017-51ab65955d7d.png)

Thank you for the help and sorry for taking your time! :)
